src/EditorFeatures/Test2/Rename/TestHost.vb

```diff
@@ -1,13 +0,0 @@
-﻿' Licensed to the .NET Foundation under one or more agreements.
-' The .NET Foundation licenses this file to you under the MIT license.
-' See the LICENSE file in the project root for more information.
-
-Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
-    Public Enum TestHost
-        InProcess
-        ' Work out of process, marshaling to/from RenameSymbolAsync
-        OutOfProcess_SingleCall
-        ' Work out of process, marshaling to/from FindRenameLocations, then marshaling to/from ResolveConflictsAsync
-        OutOfProcess_SplitCall
```

Comment:
I am not sure I understand the original semantics here. @CyrusNajmabadi to confirm if both the OutOfProcess member values are now equivalent for this to be a safe change.

Reply:
The first one wasn't used.

Reply:
Ah :-). Then no worries.

Reply:
let me see.  that doesn't sound right.

Reply:
this was used.  the rename tests are theories, and so xunit calls in with all three values.  We then check for two of htem.  i.e. we see if we're `<> InProcess` to determine if we're doing oop.  and then we check if we are in `OutOfProcess_SplitCall` to know if we're making two oop calls or one.  you are correct that we don't explicitly check for `OutOfProcess_SingleCall`.  but it's implied by the other two checks.  

Please preserve the testing.  note: you can def restructure this in some other fashoin if you want.  i.e. we could use the stadndard TestHost enum, and have a bool saying if we should do single-call or split-call.

Reply:
I see. Will fix it up.